### ISSUE TITLE
Public Sector Financial Resource codeset link could be updated

### ISSUE THREAD

**Issue by chrisdangerfield** [new-zealand-research-information-system/nzris-data-specifications#12]
The link to the Public Sector Financial Resource codeset could be updated in the Data Specifications to the [comma separated values (CSV) file on GitHub](https://github.com/new-zealand-research-information-system/nzris-codesets/blob/master/csv/public-sector-financial-resource.csv), as this file includes Budget 2019 information.

**Comment by Jason-Gush** [comment]
Good idea @chrisdangerfield 
While we're on this: how about backfilling 2009/2010, or even 2008/2009, as these are
the periods being mooted for data provision starting from.

**Comment by chrisdangerfield** [comment]
Right, yes that's a good idea and it looks like the data from the Treasury is there. I will add an issue on the codesets repository.
